hicexplorer/HiCMatrix.py

```diff
@@ -278,7 +282,7 @@ def getDekkerBins(self, fileName):
         self.header = []
         try:
             for line in gzip.open(fileName, 'r').readlines():
-                if line.startswith("#"):
+                if line.startswith(b"#"):
```

Comment:
As I understand in python 2 bytes and unicode are both `str`, but python 3 distinguishes them. However, I would expect that python 3 by default, for many situations, like reading a file, uses unicode instead bytes.